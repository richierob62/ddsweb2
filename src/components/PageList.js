import React from 'react';

const PageList = () => {
    return (
        <div className="PageList">
            <h1>This is the PageList</h1>
        </div>
    );
}

export default PageList;