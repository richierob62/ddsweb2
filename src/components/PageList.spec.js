import React from 'react';
import PageList from './PageList';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

it('renders without crashing', () => {
  const el = shallow(<PageList />);
  expect(shallowToJson(el)).toMatchSnapshot();
});

