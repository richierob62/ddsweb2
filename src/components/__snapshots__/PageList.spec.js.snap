exports[`test renders without crashing 1`] = `
<div
  className="PageList">
  <h1>
    This is the PageList
  </h1>
</div>
`;
